### ISSUE TITLE
FPWD Request for <title>

### ISSUE THREAD

**Issue by svgeesus** [w3c/transitions#410]
# Document title, URLs, estimated publication date
Incremental Font Transfer via Range Request
https://w3c.github.io/IFT/RangeRequest.html
Tues 25 Jan 2022

# Abstract
Specification for incremental font transfer via HTTP range requests.

# Status
https://w3c.github.io/IFT/RangeRequest.html#status

# Is it a delta specification intended to become a W3C Recommendation?
No

# Link to group's decision to request transition
https://www.w3.org/2022/01/18-webfonts-minutes.html#t03

# Information about implementations known to the Working Group
Not yet

**Comment by plehegar** [comment]
Approved

**Comment by plehegar** [comment]
one comment: "RangeRquest" seems very generic. Have you considered "FontRangeRequest" ? Not a blocker in any case.

**Comment by svgeesus** [comment]
Fair point on the shortname but this FPWD is being published just to trigger an exclusion opportunity. It will then be merged into the (already published) IFT spec.

So I don't see it being a problem going forward. Also, FPWD publication request was sent earlier, in advance of approval, so I would prefer not to move everything to a new directory now.

**Comment by svgeesus** [comment]
Publication requested 18 Jan, expected Tues 25 Jan

**Comment by svgeesus** [comment]
[Published](https://www.w3.org/TR/2022/WD-RangeRequest-20220125/)
